Server/src/public/stylesheets/style.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
what does the server use this css file for?

Reply:
Created ADO Item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691377